### DISCUSSION TITLE
DataTable with cell that contains array of values

### DISCUSSION THREAD

**Discussion by FruGosky** [primefaces/community#915] [PrimeReact] [answered]
Does anyone know if there is any way to achieve creating 'DataTable' with cell that contains values in array? I mean i can change the way that is displaying by passing param 'body' to the 'Column' but i think i can't implement any filtering to this because my array is converted by default to string by something similar to .join('') because from array [1,2,3] im getting cell with value of string '123' and i think i can't change behavior of it same as i can't implement filtering it anyway. Am i right or there is a some way to do this?

**Comment by melloware** [comment, marked as answer]
We chatted on Discord but the answer was to flatten your JSON structure and pass that flattened structure to the DataTable.
